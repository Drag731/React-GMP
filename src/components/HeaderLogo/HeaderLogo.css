.b-header__logo-wrap {
  color: red;
  font-weight: bold; }
