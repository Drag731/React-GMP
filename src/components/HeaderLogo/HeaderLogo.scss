.b-header {
    &__logo-wrap {
        color: red;
        font-weight: bold;
    }
}