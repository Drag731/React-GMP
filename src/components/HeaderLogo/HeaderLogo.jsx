import React from 'react';

import './HeaderLogo.css'

const HeaderLogo = () => (
    <div className="b-header__logo-wrap">
        <h1 className="b-header__logo-text">
            Filmzilla
        </h1>
    </div>
);

export default HeaderLogo;