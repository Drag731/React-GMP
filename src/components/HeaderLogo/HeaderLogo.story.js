import React from 'react';
import { storiesOf } from '@storybook/react';
import HeaderLogo from './HeaderLogo';

storiesOf('HeaderLogo', module)
    .add('with text', () => (
        <HeaderLogo />
    ));
