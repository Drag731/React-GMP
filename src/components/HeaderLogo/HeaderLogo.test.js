import React from 'react';
import renderer from 'react-test-renderer';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import HeaderLogo from './HeaderLogo.jsx'

configure({ adapter: new Adapter() });

describe('<HeaderLogo  />', () => {
    it('Renders HeaderLogo component', () => {
        const tree = shallow(
            <HeaderLogo />
        );
        expect(tree.find('div.b-header__logo-wrap')).toHaveLength(1);
    });
    it('Renders HeaderLogo component', () => {
        const tree = shallow(
            <HeaderLogo description={true}/>
        );
        expect(tree.find('.b-header__logo-link')).toHaveLength(1);
    });

});