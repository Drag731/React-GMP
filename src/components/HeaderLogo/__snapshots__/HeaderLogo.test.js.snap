// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderLogo  /> Snapshot one 1`] = `
<div
  className="b-header__logo-wrap"
>
  <h1
    className="b-header__logo-text"
  >
    Filmzilla
  </h1>
</div>
`;

exports[`<HeaderLogo  /> Snapshot two 1`] = `
<div
  className="b-header__logo-wrap"
>
  <h1
    className="b-header__logo-text"
  >
    Filmzilla
  </h1>
  <a
    className="b-header__logo-link"
    href="/filmzilla"
    onClick={[Function]}
  >
    search
  </a>
</div>
`;
